hack/setup-dev-node.sh

```diff
@@ -0,0 +1,46 @@
+ubuntu@ip-172-31-20-207:~/go/src/github.com/arktos/hack$ cat setup-dev-node.sh
```

Comment:
is this part of the script?

Reply:
It's a mistake when I copied the tested script to PR branch :(   Fixed.  Thanks.